src/main/java/seedu/address/ui/StartupCard.java

```diff
@@ -44,6 +45,12 @@ public class StartupCard extends UiPart<Region> {
     @FXML
     private FlowPane industryAndFundingStage;
 
+    @FXML
+    private FlowPane valuationDisplay;
+
+    @FXML
+    private Label valuation;
```

Comment:
Since displaying valuation is just displaying one single string, maybe you can use just label to display it.
```suggestion
    @FXML
    private Label valuation;
```